Automattic/vip-go-mu-plugins
# Search: Add X-Opaque-Id header for ES Task API

## Description
Let's send a `X-Opaque-Id` header for identifying sites. Using `vip-<site id>` as an identifier so we don't run into the partial match issue of looking for site 60 and receiving 260.

## Changelog Description

### Plugin Updated: Enterprise Search

Add `X-Opaque-Id` header to requests for ES Task API to access.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Sandbox a site with VIP Search enabled
3. `var_dump()` to see if the headers get sent
